bin/ci/ubuntu/install-dependencies.sh

```diff
@@ -9,12 +9,24 @@ do
   test -x $( type -p ${g}-$GCC_VER )
   ln -sv $(type -p ${g}-$GCC_VER) $HOME/bin/${g}
 done
-for c in clang clang++
-do
-    test -x $( type -p ${c}-$CLANG_VER )
-    ln -sv $(type -p ${c}-$CLANG_VER) $HOME/bin/${c}
-done
-export PATH=$PWD/bin:$PATH
+
+if [[ -n ${CLANG_VER:-} ]]; then
+    if [[ ! -d llvm-${LLVM_VERSION} ]]; then
+        wget -O llvm-$LLVM_VERSION.tar.xz http://llvm.org/releases/$LLVM_VERSION/clang+llvm-$LLVM_VERSION-x86_64-linux-gnu-ubuntu-14.04.tar.xz;
+        mkdir llvm-$LLVM_VERSION;
+        xzcat llvm-$LLVM_VERSION.tar.xz | tar -xvf - --strip 1 -C llvm-$LLVM_VERSION;
+    fi
+    llvm-$LLVM_VERSION/bin/llvm-config --version;
+    export LLVM_CONFIG="llvm-$LLVM_VERSION/bin/llvm-config";
+
+    for c in clang clang++
+    do
+        test -x $( type -p ${c}-$CLANG_VER )
+        ln -sv $(type -p ${c}-$CLANG_VER) $HOME/bin/${c}
+    done
+    export PATH=$PWD/llvm-$LLVM_VERSION/bin:$PWD/bin:$PATH
```

Comment:
Done. The reason I put it in both places is I'd rather have it here, but it doesn't seem to work (i.e. the script in travis uses the wrong clang if I just change the path here). I removed this and I'm just setting the path in the yml file now.